pagopa/io-app
# feat(IT Wallet): [SIW-1915] Add discovery banner in wallet onboarding screen

## Short description
This PR adds an ITW discovery banner in the wallet onboarding screen.

## List of changes proposed in this pull request
- Added `ItwDiscoveryBannerOnboarding` component
- Updated `WalletCardOnboardingScreen` with the onboarding banner

## How to test
- **Before ITW activation:** navigate to the wallet onboarding screen and verify that the feedback banner is displayed correctly.
- **After ITW activation:** navigate to the wallet onboarding scree, verify that the feedback banner is no longer displayed and ensure that the list of available credentials is displayed instead.

## Preview
<img src="https://github.com/user-attachments/assets/94777da8-c31c-4867-b8a7-4571d3aeeabf" width="200" />
